__mocks__/authenticatorMocks.ts

```diff
@@ -169,4 +169,64 @@ export class mockAuthenticatorToAutoLogin extends Authenticator {
   public requiresGetKeyConfirmation(): boolean {
     return false
   }
-}
\ No newline at end of file
+}
+
+export class mockAuthenticatorToInvalidate extends Authenticator {
```

Comment:
Since a lot of these mock classes look very similar, could we have a `MockAuthenticator` that extends `Authenticator` and then have each of these classes extend that and only override the things that are different?